giza/giza/content/newTables/inheritance.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os.path
+
+from giza.inheritance import DataContentBase, DataCache
+from giza.content.newTables.models import TableData
+
+
+class TableFile(DataContentBase):
+    content_class = TableData
+
+    def target(self, fn):
+        # fn is the source file
+        return os.path.join(self.conf.system.content.newTables.output_dir,
+                            self.conf.system.content.newTables.get_basename(fn)) + '.rst'
+
```

Comment:
each tableFile should have an attribute that defaults to None, and otherwise stores the number of columns in the table described by the file. 

If the value is none, set it to the number of column in the current doc, otherwise throw type error (in add) if current_num_columns != table_num_cloumns